clients/mobile/src/components/donation/PostCard.tsx

```diff
@@ -152,6 +167,33 @@ export const PostCard: React.FC<PostCardProps> = React.memo(({
                   </View>
                 </TouchableWithoutFeedback>
               </Modal>
+              {/* GenericModal component */}
```

Comment:
why the comment

Reply:
Bhai added that when working, but forgot to remove when committing.